js/app/modules/card/list.js

```diff
@@ -30,11 +30,20 @@ const List = new Module.Class({
             'Show synopsis', 'Whether to show the synopsis label',
             GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT_ONLY,
             true),
+        /**
+         * Property: show-context
+         * Whether to show the context label.
+         */
+        'show-context': GObject.ParamSpec.boolean('show-context',
+            'Show context', 'Whether to show the context label',
+            GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT_ONLY,
```

Comment:
I am not really sure we can do that, since in at least one case, this is determined by designers during the design of the template/app.